### DISCUSSION TITLE
New UI is a bit too Facebook and less like GitHub

### DISCUSSION THREAD

**Discussion by baggiponte** [community/community#46635] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

I really appreciate the redesign efforts, and the design team is doing a great work. However, aside from the "For you" section (whose contents I still find mostly irrelevant to me), I find the new top level banner a bit too obtrusive, especially when considering that the various update in the `Following` section are more padded than before. I feel like there is a lot less content, and the `Following` section is by far the most important medium through which I find new interesting repositories.

![image](https://user-images.githubusercontent.com/57922983/217261038-85737c67-625a-4493-a3b0-3527978411a5.png)

**Comment by bryanwweber** [comment]
Please allow us to remove this banner after I've read it! I don't mind it, but now that I've read it, I don't need to see it anymore. Thank you 😸

**Reply by spenserblack** [reply]
I was just about to say this :laughing: Yeah, seeing it once isn't a problem, but seeing it *every time I go to the homepage* can be a bit annoying. It should be dismissible, and preferably that should persist between sessions.

**Reply by Nelson-Gon** [reply]
It should preferably only be shown to new users at the log in page. Most "seasoned" users probably already know that Github is their home.
